.azure/pipelines/jobs/default-build.yml

```diff
@@ -41,217 +41,223 @@
 #
 
 parameters:
-  agentOs: 'Windows'
-  buildArgs: ''
-  configuration: 'Release'
+  agentOs: "Windows"
```

Comment:
Why switch to double-quotes? YAML supports either

Reply:
🤷‍♀ I'm fine either way. Wasn't intentional.

Reply:
Prefer we left these alone to avoid unnecessary churn